pkg/execution/state/redis_state/lua/includes/update_pointer_score.lua

```diff
@@ -2,6 +2,12 @@
 -- score.  This score should almost always be the value from `get_fn_partition_score`.
 -- It's a separate function as > 1 pointer queue may be updated at a time.
 local function update_pointer_score_to(fnID, pointerQueueKey, updateTo)
+    -- if fnID contains 
```

Comment:
To prevent enqueueing to key queues, we need to prevent existing runs from updating the account and global partition pointers for key queues. We detect this by filtering out all fnIDs starting with a curly brace, as this is guaranteed to be a fully-qualified Redis key, which is only used for key queues.

This fix should be refactored to a different level, with a proper feature toggle.

Reply:
It looks like we have introduced some sort of flag `enableKeyQueues` below — can we use a similar one here?